lwr_hw/krl/ros_position.src

```diff
@@ -0,0 +1,74 @@
+&ACCESS RVP
+&REL 3
+&PARAM TEMPLATE = C:\KRC\Roboter\Template\vorgabe
+&PARAM EDITMASK = *
+DEF ros_control( )
+;FOLD INI
+  ;FOLD BASISTECH INI
+
+    GLOBAL INTERRUPT DECL 3 WHEN $STOPMESS==TRUE DO IR_STOPM ( )
+    INTERRUPT ON 3 
+    BAS (#INITMOV,0 )
+  ;ENDFOLD (BASISTECH INI)
+  ;FOLD USER INI
+    ;Make your modifications here
+    
+
+  ;ENDFOLD (USER INI)
+;ENDFOLD (INI)
+
+;close and stop the communication
+retVal=FRISTOP()
+retVal=FRICLOSE()
+
+for i=1 to 16
+    $fri_to_int[i]=0
+    $fri_to_rea[i]=0.0
+endfor
+
+bas(#tool,1)
+;input correct tool number here and restart
+
+;stay at the same position
+ptp $pos_act_mes
+;HALT
+
+; Joint position control
+$stiffness.strategy = 10
+$stiffness.axisstiffness={ A1 1000, A2 1000, A3 1000, A4 1000, A5 1000, A6 1000, E1 1000}

```

Comment:
If this is joint position control only, you don't need this. And neither the next ones until the `$stiffness.commit=true`.